test/runtests.jl

```diff
@@ -1,55 +1,132 @@
 module TestGridapMakie
 
 using GridapMakie
+using GLMakie
 using Test
-using Makie
+using GeometryBasics
+
 using Gridap
-using Gridap.Visualization
-import AbstractPlotting; const AP = AbstractPlotting
+using Gridap.Geometry
```

Comment:
Do you need `using Gridap.Geometry` and `using Gridap.ReferenceFEs`?

Reply:
We actually do not. `using GeometryBasics` can be removed as well.